modules/user_accounts/js/permissions.js

```diff
@@ -0,0 +1,375 @@
+'use strict';
+
+var Role = React.createClass({
```

Comment:
You should be compiling this using `npm run compile` or `npm run watch`. It will produce a minified file + allow you to use JS modules, ES6 and other cool stuff